### ISSUE TITLE
When english option is coming?

### ISSUE THREAD

**Issue by winteum** [Kenshin/simpread#369]
The app looks very nice and functional. I'm wondering if an english version is in development.

**Comment by neoOpus** [comment]
cannot wait 🤤

**Comment by SMylk** [comment]
Even a simple google translate version would be godsend!

**Comment by Kenshin** [comment]
3ks comments~ 
Multilingual version will be implemented released in the future, e.g. 1.1.2 or 1.1.3

**Comment by smaragdus** [comment]
@Kenshin 

I hope that [SimpRead](http://ksria.com/simpread/) will be released in English soon. I had a peep at version **1.1.3** and it looks quite interesting and customizable but as I do not speak Chinese my use of the extension was rather limited. However the Markdown files it produced were good. It seems that [SimpRead](https://github.com/Kenshin/simpread) was removed from [Firefox](https://addons.mozilla.org/en-US/firefox/addon/simpread/), why? Is there any time line for the translation of the extension in English?

**Comment by gabrielgiordan** [comment]
Hey guys, It is a very functional and simple extension, you are awesome! An English version would be very welcome for non-Chinese speakers, hoping you release it soon.

**Comment by ysmike** [comment]
@Kenshin I've been a big fan of reader mode of Safari and your simpread comes the closest out of all the Chrome extensions out there for now. I think you'd get a lot of traction internationally if you provide an option to choose the English version at installation. Thank you & great work!

P.S. In the meantime, for us that don't know Chinese, use google translate! It works, albeit inconveniently.

**Comment by fnfontana** [comment]
Hello! I've recently installed SimpRead, looks like it's a remarkable tool based on my recent experience.
Unfortunately, I don't understand Chinese, so I can't get the best experience from it.
My native language is Brazilian Portuguese, English is my second one. 
I can volunteer to help translate both languages if my help is welcome. 

If English language is already available, though, where can I click to enable it?

**Comment by a0s** [comment]
I am writing to you from 2023. This reader plugin still looks very promising!! But I can't figure out how to switch the interface to English. Can you tell me where to click?

**Comment by Crunchbits** [comment]
Agree, looks very nifty but no English is unfortunate. Additionally adding western-type services such as PayPal to purchase the premium version would help open SimpRead to many more users.
